src/js/components/Pagination/stories/Custom.js

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+
+import { Box, Grommet, Pagination, Text } from 'grommet';
+import { grommet } from 'grommet/themes';
+import { deepMerge } from 'grommet/utils';
+import { FormPreviousLink } from 'grommet-icons/icons/FormPreviousLink';
+import { FormNextLink } from 'grommet-icons/icons/FormNextLink';
+import { ThemeContext } from 'styled-components';
+
+const customTheme = deepMerge(grommet, {
+  pagination: {
+    button: {
+      color: 'text-strong',
+      active: {
+        background: {
+          color: 'salmon',
+        },
+        color: 'text',
+      },
+      hover: {
+        background: {
+          color: 'skyblue',
+        },
+      },
+    },
+    controls: {
+      gap: 'xxsmall',
+    },
+    icons: {
+      next: FormNextLink,
+      previous: FormPreviousLink,
+    },
+  },
+});
+
+const secondaryTheme = deepMerge(grommet, {
+  button: {
+    default: {},
+    bright: {
+      color: 'text-strong',
+      border: {
+        color: 'skyblue',
+        width: '2px',
+      },
+    },
+    active: {
+      bright: {
+        background: {
+          color: '#CA9CEA',
+        },
+        border: {
+          color: 'transparent',
+        },
+        color: 'text',
+      },
+    },
+    hover: {
+      bright: {
+        background: {
+          color: 'skyblue',
+        },
+      },
+    },
+  },
+  pagination: {
+    button: 'bright',
+  },
+});
+
+export const Custom = () => (
```

Comment:
Can we please move this file under stories/typescript/Custom.tsx

This will make sure all TS definitions are working as expected?

Reply:
Done.